packages/devtools-reps/src/reps/tests/grip.js

```diff
@@ -272,11 +272,9 @@ describe("Grip - Object with uninteresting properties", () => {
     expect(getRep(object)).toBe(Grip.rep);
   });
 
-  it.skip("renders as expected", () => {
-    // @TODO This is broken at the moment.
-    // See https://bugzilla.mozilla.org/show_bug.cgi?id=1276376
+  it("renders as expected", () => {
     const renderRep = props => shallowRenderRep(object, props);
-    const defaultOutput = 'Object {c: "c", d: 1, a: undefined, more...}';
+    const defaultOutput = 'Object { a: undefined, c: "c", d: 1, more... }';
```

Comment:
I changed the order because I read that the optimal result required was (https://bugzilla.mozilla.org/show_bug.cgi?id=1276376):
![screenshot_1](https://user-images.githubusercontent.com/36955296/45539940-74e33b00-b80b-11e8-8b2e-b2353786060a.png)
But maybe I understood wrong.

I tried `...` but it fail.